src/IniFile.cpp

```diff
@@ -191,6 +193,7 @@ bool cIniFile::WriteFile(const AString & a_FileName) const
 	// Normally you would use ofstream, but the SGI CC compiler has
 	// a few bugs with ofstream. So ... fstream used.
 	fstream f;
+	AString runvalue, writevalue;
```

Comment:
Not sure what the point of having two separate strings here is - since writevalue is always set to the value of runvalue! Same for rawvalue up above.

Please remove one.